File: c/9.c
/*
    main 함수 안의 내용만 수정해주세요.
    수정할 경우, 자동 채점 프로그램이 제대로 동작하지 않을 가능성이 있습니다.
*/

#include <stdio.h>

int main() {
    // 이곳에 코드를 작성해주세요!
    int n, i, ans = 0;
    
    do{
        scanf("%d",&n);
        if(n <= 0){
            printf("X\n");
        }
    }while(n <= 0);

    for(i = 1; i <= n; i++){
        ans += i;
    }
    printf("%d\n",ans);
    

    return 0;
}

File: python/9.py
"""
    main 함수 안의 내용만 수정해주세요.
    수정할 경우, 자동 채점 프로그램이 제대로 동작하지 않을 가능성이 있습니다.
"""

def main():
    # 이곳에 코드를 작성해주세요!

    while True:
        n = int(input())
        if n <= 0 :
            print('X')
        else:
            break
    

    ans = 0
    for i in range(n):
        ans += i + 1
    print(ans)

    return


if __name__ == '__main__':
    main()
